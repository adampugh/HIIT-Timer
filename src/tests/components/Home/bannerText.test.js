import React from "react";
import { shallow } from "enzyme"; 

import BannerText from "../../../components/Home/bannerText";

test("should render bannerText correctly", () => {
    const wrapper = shallow(<BannerText />);
    expect(wrapper).toMatchSnapshot();
});