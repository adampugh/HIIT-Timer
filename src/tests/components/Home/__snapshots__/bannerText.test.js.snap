// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render bannerText correctly 1`] = `
<div
  className="bannerText"
>
  <div
    className="bannerText__container"
  >
    <h2>
      Create custom high intentsity interval workouts with HIIT Timer. Get in shape and loose weight with HIIT Timer today!
    </h2>
    <img
      alt="arm icon"
      className="bannerText__image"
      src="dumbbell-icon.png"
    />
  </div>
</div>
`;
